sw/device/tests/clkmgr_jitter_frequency_test.c

```diff
@@ -36,6 +49,47 @@ enum {
 static dif_clkmgr_t clkmgr;
 static dif_pwrmgr_t pwrmgr;
 
+// Test with thresholds for jitter enabled expecting no failures, and then
+// with thresholds for jitter disabled expecting failures.
+static void test_clock_frequencies_with_jitter_enabled(uint32_t delay_micros) {
+  LOG_INFO("Testing frequencies with jitter enabled");
+  CHECK_STATUS_OK(clkmgr_testutils_enable_clock_counts_with_expected_thresholds(
+      &clkmgr, /*jitter_enabled=*/true, /*external_clk=*/false,
+      /*low_speed=*/false));
+  busy_spin_micros(delay_micros);
+  // This checks there are no errors.
+  CHECK_STATUS_OK(clkmgr_testutils_check_measurement_counts(&clkmgr));
+  CHECK_STATUS_OK(clkmgr_testutils_disable_clock_counts(&clkmgr));
+  if (kDeviceType == kDeviceSimDV || kDeviceType == kDeviceSilicon) {
+    // Set thresholds for jitter disabled expecting failures.
+    CHECK_STATUS_OK(
+        clkmgr_testutils_enable_clock_counts_with_expected_thresholds(
+            &clkmgr, /*jitter_enabled=*/false, /*external_clk=*/false,
+            /*low_speed=*/false));
+    busy_spin_micros(delay_micros);
+    dif_clkmgr_recov_err_codes_t err_codes;
+    CHECK_DIF_OK(dif_clkmgr_recov_err_code_get_codes(&clkmgr, &err_codes));
+    CHECK(err_codes != 0);
+  }
```

Comment:
Should we add a print here for  FPGA platforms to highlight that this part of the test is skipped?

Reply:
Done.